tests/testthat/test-update_snapshot.R

```diff
@@ -85,37 +76,177 @@ test_that("update_snapshot() works", {
     if (inherits(conn, "SQLiteConnection")) {
       types <- types |>
         purrr::map_if(~ identical(., "POSIXct"), "character") |> # SQLite does not support POSIXct
-        purrr::map_if(~ identical(., "logical"), "numneric") |>  # SQLite does not support logical
+        purrr::map_if(~ identical(., "logical"), "numeric") |>  # SQLite does not support logical
         as.character()
     }
 
     checkmate::expect_data_frame(logs, nrows = 1, types)
 
+    # Check the content of the log table
+    expect_identical(as.character(logs$date), as.character(timestamp))
+
+    db_table_id <- id(db_table, conn)
+    if ("catalog" %in% colnames(logs)) expect_identical(logs$catalog, purrr::pluck(db_table_id, "name", "catalog"))
+    expect_identical(logs$schema, purrr::pluck(db_table_id, "name", "schema"))
+    expect_identical(logs$table, purrr::pluck(db_table_id, "name", "table"))
+
+    expect_identical(logs$n_insertions, 32L)
```

Comment:
Here we condition on mtcars never changing ... is it safe or should we use nrow(.data)?